content/models/optimal_steps_model/index.md

```diff
@@ -0,0 +1,152 @@
+
+The obstacle potential $P_{o,j}(x)$ is defined as 
+
+<!-- $$ -->
+<!-- \begin{eqnarray} -->
+<!-- P_{o,j}(x) = \left\{ -->
+<!-- \begin{array}{ll} -->
+<!-- \delta_1(x) + \delta_2(x) & d_{o,j}(x) < r_l \\\ -->
+<!-- \delta_1(x)  & r_l \leq d_{o,j}(x) < w_o \\\ -->
+<!-- 0 & \, \textrm{otherwise,} \\\ -->
+<!-- \end{array} -->
+<!-- \right. -->
+<!-- \end{eqnarray} -->
+<!-- $$ -->
+
+$$
+P_{o,j}(x) =
+\begin{array}{ll}
+\delta_1(x) + \delta_2(x) & d_{o,j}(x) < r_l \\\
+\delta_1(x)  & r_l \leq d_{o,j}(x) < w_o \\\
+0 &  \textrm{otherwise,} \\\
+\end{array}
+$$
+where $d_{o,j}(x)$ is the distance to the closest point of the obstacle from position $x$, $w_o$ defined the width of the obstacle repulsion, $r_l$ is the torso radius of agent $l$. The functions $\delta_i$ are defined as
+$$
+\begin{align}
+\delta_1(x) &= 6 \exp{\left[ 2 \left( \left(  \frac{d_{o,j}(x)} {w_o} \right)^{2} - 1\right)^{-1} \right]}\;\text{and}\\\
+\delta_2(x) &= 10^5 \exp{\left[ \left( \left(  \frac{d_{o,j}(x)} {r_l} \right)^{2} - 1\right)^{-1} \right]}. \\
+\end{align}
+$$
+The repulsion between two agents is achieved by the distance-dependent potential function $P_{p,i}$. In Vadere, it is implemented as
+
+$$
+P_{p,i}(x) = 
+\begin{array}{ll}
+\phi_1(d_{l,i}(x)) + \phi_2(d_{l,i}(x)) + \phi_3(d_{l,i}(x)) & d_{l,i}(x) <r_i + r_l, \\\
+\phi_1(d_{l,i}(x)) + \phi_2(d_{l,i}(x)) & r_i + r_l \leq d_{l,i}(x) < w_{int}  +r_i+r_l, \\\
+\phi_1(d_{l,i}(x))  & w_{int} + r_i + r_l \leq  d_{l,i}(x) < w+r_i+r_l\\\
+0, & \textrm{otherwise,} \\\
+\end{array}
+$$
+
+where $d_{l,i}(x)$ is the Euclidean distance between agent $l$  and agent $i$. $r_i, r_l$ are the respective torso radii of agent $i,l$. $w_{int}$ and  $w$ are the intimate and the personal space width according to Hall. The functions $\phi_i$ are defined as
+$$
+\begin{align}
+\phi_1({d_{i,j}}) &= \mu \exp{\left[ 4 \left( \left(  \frac{d_{i,j}} {w +r_i +r_l} \right)^{2} - 1\right)^{-1} \right]} \\\
+\phi_2({d_{i,j}}) &= \frac{ \mu } {a_p} \exp{\left[ 4 \left( \left(  \frac{d_{i,j}} {w_{int} +r_i +r_l} \right)^{2} - 1\right)^{-1} \right]} \\\
+\phi_3({d_{i,j}}) &= 10^3 \exp{\left[ \left( \left(  \frac{d_{i,j}} {r_i +r_l} \right)^{2} - 1\right)^{-1} \right]}
+\end{align}
+$$
+
+The potential function is based on Hall's theory of interpersonal distances which describes four distance zones around a person [[4]](#Hall). Accordingly, the potential function is defined piece-wise on rings around each agent: a circular core for collision avoidance, a first ring that represents the intimate space, and a second ring that represents personal space. Agents outside the personal zone do not affect other agents' path choice. This is mathematically modeled by setting the  the potential function to zero.
+
+
+{{< figure src="figure3.png" caption="Agent's potential versus distance." >}}
```

Comment:
This figure might need more Explanation. There are two functions showing seemingly the same thing. What is the difference?